### DISCUSSION TITLE
Lock and login screen as out of the box product

### DISCUSSION THREAD

**Discussion by Dregorio1** [hyprwm/Hyprland#4295] [General]
Hello, is there plan in near future to make for hyprland "the whole" ecosystem, like dedicated lock screen and login screen (something like ubuntu has)? Right now we have to install like swaylock and something else for loging. It would be nice to have everything out of the box.

**Comment by MightyPlaza** [comment]
that isn't the job of a window manger (see https://github.com/hyprwm/Hyprland/issues/3475)

there are already projects do something similar like https://garudalinux.org/ or https://nwg-piotr.github.io/nwg-shell/
